contractcourt/briefcase.go

```diff
@@ -1008,6 +1045,139 @@ func (b *boltArbitratorLog) FetchChainActions() (ChainActionMap, error) {
 	return actionsMap, nil
 }
 
+// LogLocalForceCloseInfo records the passed-in localForceCloseInfo
+// object within the ArbitratorLog for deferred consumption. This is
+// typically invoked when the channel force closure is first initiated
+// either by the user or automatically (link error, HTLC actions, etc.).
+func (b *boltArbitratorLog) LogLocalForceCloseInfo(
+	info localForceCloseInfo) error {
+
+	return kvdb.Update(b.db, func(tx kvdb.RwTx) error {
+		scopeBucket, err := tx.CreateTopLevelBucket(b.scopeKey[:])
+		if err != nil {
+			return err
+		}
+
+		b := new(bytes.Buffer)
+
+		// first byte is userInitiated bool
+		err = binary.Write(b, endian, info.userInitiated)
```

Comment:
We should use TLV for any new data structure that we save to the DB. That will make it way easier to extend this info structure as well. See the example in `fetchChanInfo()` in `channeldb/channel.go`.